app/src/gui/components/notebook/settings/index.tsx

```diff
@@ -126,14 +130,14 @@ export default function NotebookSettings(props: {uiSpec: ProjectUIModel}) {
                   label={<Typography>{isSyncing ? 'On' : 'Off'}</Typography>}
                 />
                 <Typography variant={'body2'}>
-                  This control is notebook and device specific. If this option
-                  is enabled, Fieldmark™ will automatically download and show
-                  images and attachments created by other devices. Be aware that
-                  this may be resource intensive and use your mobile data plan.
-                  Disable this setting to minimise network usage. This setting
-                  will not affect uploading of your data from this device to the
-                  central server. Attachments are always uploaded to the server
-                  regardless of this setting.
+                  This control is {NOTEBOOK_NAME} and device specific. If this
+                  option is enabled, Fieldmark™ will automatically download and
```

Comment:
@stevecassidy what do you think about branding here given its apache 2.0 it should be appropriate for the BSS to theme out Fieldmark (other than in attribution statements like "Thanks to ARDC for funding Electronic Field Notebooks").  So this could say "the BSS will automatically download..." or similar.

Reply:
But we can do that in separate PR.

Reply:
This can just be 'the app'.